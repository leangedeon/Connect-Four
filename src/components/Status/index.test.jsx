import React from 'react';
import { shallow } from 'enzyme';
import { Status } from './';


describe('Status', () => {
    it('should match snapshot', () => {
        
        const wrapper = shallow(<Status />);

        expect(wrapper).toMatchSnapshot();
    });
});