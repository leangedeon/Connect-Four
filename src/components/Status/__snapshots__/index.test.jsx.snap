// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status should match snapshot 1`] = `
<styled.div>
  <div
    className="d-grid gap-2"
  >
    <button
      className="btn btn-primary"
      type="button"
    >
      State: 
    </button>
  </div>
</styled.div>
`;
